compiler-plugin/src/main/scala/com/trueaccord/scalapb/compiler/ProtobufGenerator.scala

```diff
@@ -382,8 +382,14 @@ class ProtobufGenerator(val params: GeneratorParams) extends DescriptorPimps {
       .add("}")
       .add("final override def serializedSize: Int = {")
       .indent
-      .add("if (__serializedSizeCachedValue == -1) { __serializedSizeCachedValue = __computeSerializedValue() }")
-      .add("__serializedSizeCachedValue")
+      .add("var read = __serializedSizeCachedValue")
```

Comment:
can you get rid of 'read' and use '__serializedSizeCachedValue` directly instead?